TGC/events/Filibuster.txt

```diff
@@ -0,0 +1,623 @@
+### Filabusters ###
+
+	## William Walker invades Nicaragua
+		country_event = {
+			id = 8870780
+			title = "EVTNAME8870780"
+			desc = "EVTDESC8870780"
+			picture = "Cubanrebels"
+
+			fire_only_once = yes
+			trigger = {
+				tag = NIC
+				year = 1854
+				NOT = { year = 1860 }
+			}
+
+			mean_time_to_happen = {
+				days = 286
+			}
+
+			option = {
+				name = "EVTOPTA8870780"
+
+				prestige = -5
+				NIC = {
+					any_owned = {
```

Comment:
Will this event make sense if Nicaragua owns neither of these two provinces, or if it doesn't own 2198 and then pops from 188 move to a place not in the Filibuster state?